server/sockets/socket-server.js

```diff
@@ -55,10 +55,42 @@ io.on('connection', (socket) => {
     if (people.length > 1) {
       people.allowed = 1;  // setting var for game started true
       io.in(data.roomname).emit('usersList', usernames)       // emitting usernames list to clients
+      let players = people.length                     //no of players
+      //shuffled cards array
+      for (var a=[],i=0;i<54;++i) a[i]=i;
+  
+      function shuffle(array) {
+        var tmp, current, top = array.length;
```

Comment:
Please don't use `var`

Reply:
need to use it const give an linting error in ++i and let gives an error in a =shuffle(a).Any other suggestions?